Move away from FD, detail how we use our PM system (#1335)
ferrocene/ferrocene: ferrocene/doc/qualification-plan/src/development.rst

.. _dev-phase-design:

> Dev Phase 1: Development Ticket Approval
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Dev Phase 1: Development Ticket Approval
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

To ensure adequate quality, changes that are deemed *major* by the authors or
reviewers need to have a development ticket written, approved before
development can begin.

Whether a change is deemed major is at the discretion of the Ferrocene
developers and reviewers assigned to the task. Still, any member of the

> Ferrocene team can ask for a change to be deemed major (and thus require an

Reviewer: ```suggestion
Ferrocene team can ask for a change to be deemed major (and thus require a
```

ticket), even if the developers and reviewers decided it was minor.

Tickets must be created in the project management tool under the respective

> project board, it must be appropriately scoped and detail acceptance criteria.

Reviewer: ```suggestion
project board, they must be appropriately scoped, and they must detail acceptance criteria.
```

Large tickets must be broken down into smaller, independent tickets. A ticket

> must be approved by at least another member of the Ferrocene team without any

Reviewer: ```suggestion
must be approved by at least one other member of the Ferrocene team without any
```
